server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -167,17 +168,35 @@ public HttpResponse grantToken(
             String cookieTimeout =
                 ServerProperties.getInstance().getProperty("server.cookie-timeout", "P5D");
             Cookie cookie =
-                Cookie.secureBuilder(AuthDecorator.UC_TOKEN_KEY, accessToken)
-                    .path("/")
-                    .maxAge(Duration.parse(cookieTimeout).getSeconds())
-                    .build();
+                createCookie(
+                    AuthDecorator.UC_TOKEN_KEY,
+                    accessToken,
+                    "/",
+                    Duration.parse(cookieTimeout).getSeconds());
             responseHeaders.add(HttpHeaderNames.SET_COOKIE, cookie.toSetCookieHeader());
           }
         });
 
     return HttpResponse.ofJson(responseHeaders.build(), response);
   }
 
+  @Get("/logout")
+  public HttpResponse logout(HttpRequest request) {
+    return request.headers().cookies().stream()
+        .filter(c -> c.name().equals(AuthDecorator.UC_TOKEN_KEY))
+        .map(Cookie::value)
```

Comment:
You don't need this anymore.

Reply:
Updated and also changed  logout to post